pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -20,16 +20,16 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"time"
+
+	"golang.org/x/net/context"
```

Comment:
why this package and not the stdlib package?

Reply:
This patch didn't change the imports, just re-ordered so I think this is ok. But good question, we might want to change that(?)